pgmpy/inference/approximate_inference/Sampling.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+import networkx as nx
+
+from pgmpy.models import BayesianModel
+from pgmpy.utils.mathext import sample_discrete
+from pgmpy.inference import Inference
+
+
+class BayesianModelSampling(Inference):
+    """
+    Class for sampling methods specific to Bayesian Models
+
+    Parameters
+    ----------
+    model: instance of BayesianModel
+        model on which inference queries will be computed
+
+
+    Public Methods
+    --------------
+    forward_sample(size)
+    """
+    def __init__(self, model):
+        if not isinstance(model, BayesianModel):
+            raise TypeError("Argument must be of type BayesianModel")
```

Comment:
@pratyakshs I feel a better message can be `model must be an instance of BayesianModel`. What do you think?